### DISCUSSION TITLE
SV3D works fine through video box in generation tab.

### DISCUSSION THREAD

**Discussion by gonduras** [Stability-AI/StableSwarmUI#287] [Show and tell]
just throw the models inside models folder

**Comment by rhd2330** [comment]
All my Lora's disappeared on Swarm.

All my Lora's disappeared, and the dropdown settings below it had disappeared. I don't know what I did; I was messing around with the depth settings, and everything disappeared. I have tried to get it back, but nothing I have done is working. This is the error it is showing me:

Error: Invalid value '' for parameter 'depth', must be type 'Int32'

I have not gotten it fixed yet, but if you would like to try to do what I think I did, I wouldn't advise it, but maybe you want to try. Everything else works fine; it's just everything under the Lora's disappears, and I mean everything.

In the depth field, type 1.1; that is what I think I did, and everything on Swarm vanished under Loras.

You may say, "Just type in a different value," but the place where you type in a different value is gone. There is nothing there. Everything else is working. If there is another location to get to it, please let me know. Thank you.

**Reply by mcmonkey4eva** [reply]
You're in the wrong place, you want https://github.com/mcmonkeyprojects/SwarmUI or the Discord https://discord.gg/q2y38cqjNw
